Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine/git:v2.30.0
+
+RUN mkdir /ghcli
+WORKDIR /ghcli
+RUN wget https://github.com/cli/cli/releases/download/v1.5.0/gh_1.5.0_linux_386.tar.gz -O ghcli.tar.gz
+RUN tar --strip-components=1 -xf ghcli.tar.gz
+RUN mv bin/gh /bin/
+
+ENTRYPOINT ["/bin/ash"]
```

Comment:
What does this do?

Reply:
Sorry for that, this should not be here at all ! I've removed it